ios/SpectrumKitSample/SpectrumKitSample-iOS/ConfigurationViewModel+Options.swift

```diff
@@ -88,6 +88,61 @@ extension ConfigurationViewModel {
 
     static var allValues: [DefaultBackgroundColor] = [.black, .white, .blue, .red, .green]
   }
+    
+  enum CompressionLevel: Int, AlertOptions {
+    case `default`
+    case none
+    case bestSpeed
+    case bestCompression
+    
+    var rawValue: Int {
+      switch self {
+      case .default:
+        return PngCompressionLevelDefault
+      case .none:
+        return PngCompressionLevelNone
+      case .bestSpeed:
+        return PngCompressionLevelBestSpeed
+      case .bestCompression:
+        return PngCompressionLevelBestCompression
+      }
+    }
+    
+    init?(rawValue: Int) {
+      switch rawValue {
+      case PngCompressionLevelNone:
+        self = .none
+      case PngCompressionLevelBestSpeed:
+        self = .bestSpeed
+      case PngCompressionLevelBestCompression:
+        self = .bestCompression
+      default:
+        self = .default
+      }
+    }
+    
+    // MARK: AlertOptions
+    
+    var title: String {
+      switch self {
+      case .default:
+        return "Default"
+      case .none:
+        return "None (\(rawValue))"
+      case .bestSpeed:
+        return "Best speed (\(rawValue))"
+      case .bestCompression:
+        return "Best compression (\(rawValue))"
```

Comment:
nit: Would be great to use `NSLocalizedString` for user-facing strings. Something like:

```
let formatString = NSLocalizedString("%@ (%d)", comment: "Png Compression Format String")
return String.localizedStringWithFormat(formatString, self.title, self.rawValue)
```